.tekton/tasks/publish-reg.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
I think this task can be assumed to exist on the cluster already, in the automotive-dev-operator-system namespace
similar to https://github.com/rh-sdv-cloud-incubator/aib-devspaces/blob/be9ddf6a5eb092b7ed5fb9c25bc63fcdaf3c0d51/.tekton/push.yaml#L103

Reply:
looks like it